### ISSUE TITLE
esbuild: importing from jsr: via deno.json mapping inserts an extra slash

### ISSUE THREAD

**Issue by domenic** [lumeland/lume#706]
### Version

2.4.3

### Platform

Windows

### What steps will reproduce the bug?

I want to use the pattern [encouraged by the official Deno docs](https://docs.deno.com/runtime/fundamentals/standard_library/#importing-standard-library-modules) of using specifiers like

```js
import { escape as htmlEscape } from "@std/html/entities";
```

and then in `deno.json` mapping `@std/html` to `jsr:@std/html@^1.0.3`.

However, when Lume with the esbuild plugin compiles these files, it accidentally inserts an extra slash: the output is

```js
import { escape as htmlEscape } from "https://esm.sh/jsr//@std/html@^1.0.3/entities";
```

### How often does it reproduce? Is there a required condition?

Full repro:

**deno.json**

```json
{
  "imports": {
    "@std/html": "jsr:@std/html@^1.0.3",
    "lume/": "https://deno.land/x/lume@v2.4.2/"
  },
  "tasks": {
    "lume": "echo \"import 'lume/cli.ts'\" | deno run -A -",
    "build": "deno task lume",
    "serve": "deno task lume -s"
  },
  "compilerOptions": {
    "types": [
      "lume/types.ts"
    ]
  }
}
```

**_config.ts**

```ts
import lume from "lume/mod.ts";
import esbuild from "lume/plugins/esbuild.ts";

const site = lume({
  src: "./src",
});

site.copy([".html"]);

site.use(esbuild({
  extensions: [".ts"],
  options: {
    bundle: false,
    minify: false,
    treeShaking: false,
    keepNames: false,
  },
}));

export default site;
```

**src/index.html**
```html
<!DOCTYPE html>
<meta charset="utf-8">
<title>Test</title>

<script type="module" src="test.js"></script>
```

**src/test.ts**
```ts
import { escape as htmlEscape } from "@std/html/entities";

document.body.innerHTML = htmlEscape("Hello <from Lume>!");
```

### What is the expected behavior?

The output in `_site/test.js` should be something like

```js
import { escape as htmlEscape } from "https://esm.sh/jsr/@std/html@^1.0.3/entities";
document.body.innerHTML = htmlEscape("Hello <from Lume>!");
```

### What do you see instead?

The output in `_site/test.js` is instead

```js
import { escape as htmlEscape } from "https://esm.sh/jsr//@std/html@^1.0.3/entities";
document.body.innerHTML = htmlEscape("Hello <from Lume>!");
```

### Additional information

This is relatively important if one has code that is running both as part of the build process and in the built output.

**Comment by oscarotero** [comment]
That's a good catch. I'll fix it in the next version. Thanks!

**Comment by oscarotero** [comment]
This is already fixed and will be available in Lume 2.5.
If you want to use it now, you can upgrade Lume to the latest development version with `deno task lume upgrade --dev`.
